README.md

```diff
@@ -32,12 +32,12 @@ Additional background: [prior discussion](https://github.com/tc39/ecma402/issues
 Units of measurement can be formatted as follows:
 
 ```javascript
-(9.81).toLocaleString("en-US", {
+(299792458).toLocaleString("en-US", {
     style: "unit",
     unit: "meter-per-second",
```

Comment:
this notation is correct following `UTS #35`, but sounds like other parts of the Readme is using `unit: "speed-meter-per-second"` which sounds incorrect and inconsistent.

Reply:
Good catch, fixed